packages/scenes-react/src/components/VizPanel.test.tsx

```diff
@@ -55,4 +55,32 @@ describe('VizPanel', () => {
 
     expect(scene.state.children.length).toBe(0);
   });
+
+  it('Should render with titleItems', () => {
+    const scene = new SceneContextObject();
+    const viz = VizConfigBuilders.timeseries().build();
+    const titleItems = <div>Title Item</div>;
+
+    const { getByText } = render(
+      <TestContextProvider value={scene}>
+        <VizPanel title="Test" titleItems={titleItems} viz={viz} />
```

Comment:
I'm unfamiliar with how the VizPanel component is supposed to be used, but because titleItems is a PanelChrome prop, would it need to be like this?

```suggestion
         <PanelChrome titleItems={titleItems}>
            <VizPanel title="Test"  viz={viz} />
        </PanelChrome>
```